Rectangle/de.lproj/Main.strings

```diff
@@ -441,7 +441,7 @@
 "lgE-cR-cQ5.title" = "Gehen Sie zu Systemeinstellungen → Sicherheit → Datenschutz → Bedienungshilfen";
 
 /* Class = "NSTextFieldCell"; title = "When the menu bar icon is hidden, relaunch Rectangle from Finder to open the menu."; ObjectID = "ltc-mf-BHr"; */
-"ltc-mf-BHr.title" = "Um das Menü zu öffnen, wenn das Symbol in der Menüleiste ausgeblendet ist, starten Sie Rectangle im Finder neu.";
+"ltc-mf-BHr.title" = "Starte Rectangle erneut vom Finder aus, um das Menü zu öffnen, wenn das Menüleistensymbol ausgeblendet ist.";
```

Comment:
I need to update this in the localization files, but this has changed to:
"When the menu bar icon is hidden, relaunch Rectangle from Finder to open."

(the functionality changed in a recent version)

Would you mind translating that in this PR?

I'll update the localization files in bulk after this PR is done and merged.

Reply:
Hi, 
I translated it now. Best :)